Devscord.DiscordFramework.UnitTests/Commands/RunnerOfIBotCommandMethodsTests/Tests/DefaultCommandsTests.cs

```diff
@@ -116,6 +116,7 @@ public async Task ShouldGiveULongNumber(string message, ulong expectedULong)
         }
 
         [Test]
+        [SetCulture("pl-PL")]   // for Github tests. The tests run right only for polish culture 
         [TestCase("-double -testdouble 8", 8.0)]
         [TestCase("-double -testdouble       -103", -103.0)]
         [TestCase("-double     -testdouble      23,998   ", 23.998)]
```

Comment:
a jakby tak zrobić, że zarówno przecinek jak i kropka robią to samo, czyli oddzielają część dziesiętną?
wtedy niezależnie od kultury byłoby to samo

Reply:
1. Po co tak robić, skoro bot działa na serwerze, gdzie są same polaki, które wiadomo raczej na pewno będą używać przecinka.
2. W kulturze amerykańskiej przecinek może też służyć jako separator do dużych liczb
3. Poza tym, po co tak kombinować skoro obecnie bot nie posiada żadnej komendy w której jest wymagane podanie liczby. Jeśli byłaby taka potrzeba to po prostu się doda taką "uniwersalną" funkcjonalność.

Reply:
1. bot miał być uniwersalny, dlatego wszystkie komunikaty można przetłumaczyć na inne języki dzięki trzymaniu responses w bazie
2. tak, choć rzadziej się to używa, niż seperatora części dziesiętnej
3. mute? można podać np. 1,5h

Reply:
1. No cóż, uniwersalność trochę kosztuje, obecny "mechanizm" jest dosyć prosty, po prostu trzeba wpisywać liczby na podstawie formatu w jednej kulturze. A gdybym chciał to zrobić uniwersalnie, musiałbym trochę kombinować. Np: Musiałbym na podstawie wprowadzonej liczby stwierdzić co to za kultura i po sprawdzeniu trzeba byłoby przetwarzać daną liczbę pod kątem wykrytej kultury. Na stacku widziałem takie rozwiązanie tego problemu: https://stackoverflow.com/questions/15616341/detecting-culture-info-from-a-string-which-has-numeric-value
Szczerze to widać, że zbyt fajnie nie da się tego zrobić. Ewentualnie przyszło mi do głowy, że kulturę mógłby określać użytkownik, ale trochę jest to dziwne rozwiązanie. 

Myślę, że jak będzie potrzeba zmiany to się po prostu zmieni się założenia i uczyni się ten mechanizm bardziej uniwersalny, bo obecnie nie ma takiej potrzeby. 

PS. Tak się zastanawiam, bo u mnie domyślnie program ma kulturę polską, a pytanie jaka kultura będzie na produkcji, bo obecnie jest to sprawdzane pod kątem kultury domyślnej w aplikacji. 
2. to zależy...
3. Nie do końca, bo mute trochę inaczej działa niż to, w zasadzie to jest oddzielny mechanizm i obecnie taki zapis z przecinkiem, bot uznałby za błędny format.